.github/branch-deploy-config-map.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
```

Comment:
I wonder with these `.github/branch-deploy-*` files, could we put them all in a `.github/branch-deploy/` directory to keep them organised in that way? Might be slightly nicer, but not a big deal.

Reply:
This is a good idea, but we can't do it :( https://github.community/t/feature-request-allow-workflow-configuration-in-sub-folders/16776

Reply:
Wait, I've misunderstood, I'll move the kubectl config files